rules/windows/process_creation/proc_creation_win_susp_office_token_search.yml

```diff
@@ -0,0 +1,21 @@
+title: Suspicious Office Token Search Via CLI
+id: 6d3a3952-6530-44a3-8554-cf17c116c615
+status: experimental
+description: Detects possible search for office tokens via CLI by looking for the string "eyJ0eX"
+author: Nasreddine Bencherchali
+references:
+    - https://mrd0x.com/stealing-tokens-from-office-applications/
+date: 2022/10/25
+tags:
+    - attack.credential_access
+    - attack.t1528
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains: 'eyJ0eX'
```

Comment:
Please try to find a bigger encoded substring

Reply:
After thinking about it. I think it's better to leave the short string for now since it's the one used by the tools/technique we know.

A long string while it reduces the FP, it might not be as useful. As people using this will most likely use it as it was described in the blog or use one of the available tools.

I added a longer version as a comment that we or anyone can use if a lot of FP are encountered.

Let me know what you think.

Reply:
Yes, that's a good argument. We could add both hashes. The longer one as is and the shorter one with surrounding spaces/`"`/`'`. That should cover all the cases, shouln't it?

Reply:
Good idea. I pushed the change.